.bit/responses/5.6-Week 5 Step 6.md

```diff
@@ -0,0 +1,32 @@
+---
+files: n/a
+stepType: checks
+scripts: n/a
+week: 5
+step: 6
+name: Week 5 Step 6
+---
+Week 5 Step 6 ◯◯◯◯◯⬤ | 🕐 Estimated completion: 25 minutes
+
+## Documentation
+
+### ✅  Task:
+
+- [ ] Document your first component in blog.md
+- [ ] Commit your changes in the `week5` branch
+
+Once you have successfully completed developing your first component, you can start documenting your project blog. Remember that the purpose of this blog is to help someone else create your project from scratch, so as you start writing, you will need to break down your code as much as possible and explain it so that someone else can learn from your example.
+
+### Quick tips
+
+* If your project required any kind of groundwork like installing npm packages or setting up Azure locally, document this first in "Step 0"
+* Break down your code:
+  * Copy all of your code and paste it into a code snippet in your blog
+  * Break it down into multiple snippets in order of how you wrote the code
+  * Before each snippet, write a description of what the code is for and how it works
+  * Define keywords that were new to you or would be to your audience
+  * Tip: use substeps to make your blog easier to follow
+
+If you have completed multiple components of your project this week, document them as you go.
+
+## **:camping: To move on, commit your code in a branch called "week5", update your blog, and comment a summary of what you have developed so far!**
```

Comment:
We've removed the "to move on" section, all of this will go in the checklist at the top